### DISCUSSION TITLE
Real Yield > Potemkin Yield. Thoughts on P2P Capital Markets

### DISCUSSION THREAD

**Discussion by humansinstitute** [GaloyMoney/galoy#1361] [Ideas]
Hi All,

I've been discussing with Andrew and Kemal some ideas around different services which could leverage a community banking model. 

The base idea being, what services would become possible in a world in which you have 1000s of independent community banks operating in a country on an open monetary standard. 

One area I've been keen to explore in Australia is how you could use this approach to free up capital markets and more efficiently channel funds into robust, sustainable and profit generating SMEs and local businesses - however, it's not entirely clear I could legally do this here, weirdly :) 

But that's not necessarily true of El Salvador and other areas that will come online making bitcoin legal tender and looking to incentivize foreign investment.  So it seems like a better use of time to think through how this could be deployed there and we can always pick it up elsewhere when the time is right :) 

Whilst there is a good degree of "rah rah" around volcano bonds (and who doesn't want a little more freedom from the IMF), a centrally planned and issued investments structure has never felt like the most "bitcoin" way to open up the country to the transformative power of international finance.

So the purpose of this idea is to explore an alternative, bottom up, market driven approach to international investment. 

This would leverage the network of local community banks to assist in mitigation of credit risk and make international investments safer and more appealing to a global audience in order to drive yield on investments.

This audience can then discover and invest in these concerns, in return for yield on an ongoing basis.  

However, this would be real yield from profitable businesses not ponzi Potemkin yield from defi shitcoins. Maybe that's what that acronym was supposed to mean all along. 

My intent is to work on a design spec, mock ups, process maps etc to explore the design space for peer to peer capital markets and how this could be implemented.

Specifically I think it will be important to clarify the different trade offs and strategies available to mitigate the default risk inherent in foreign investment without the usual guarantees of government enforcement exploring options like streaming investments, revenue splits from merchant terminals, fidelity bonds on the behalf of the community bank, leveraging local bank knowledge etc. 

The plan is to develop the work here: https://github.com/humansinstitute/BitcoinForHumans/blob/master/CommunityBanking/commbank_p2p_cap.md

Any comments on best ways to document this or general ideas on the concept are all appreciated :)

**Comment by tanner-stone** [comment]
I read the [breakdown](https://github.com/humansinstitute/BitcoinForHumans/blob/master/CommunityBanking/commbank_p2p_cap.md) and think this is a really cool idea. Full disclosure I'm not a dev or even in Bitcoin professionally, but I've been following/learning about it for a year and have some thoughts. 

The biggest risk you cite in the description is the business defaulting on the loan or intentional theft of the investment. I wonder if this can be solved with multisig and programmable traunches - details below:
- In traditional finance, disbursements to investors occur in an order that reflects how secure the terms of the contract are (i.e. senior debt, mezzanine debt, preferred equity, then common equity). 
- If a multisig wallet could be created between the investor, community bank, and business that has a script with rules to disburse funds in an order based on the contract terms (prioritizing security/seniority), coupon payments and/or dividends could be automated in a method that all parties agree to.
- To assure the investor of honest payments, there needs to be some confirmation that the wallet(s) the business is sending disbursements from represents the business's actual revenue and/or profits (depending on the contract) and will have sufficient funds. If not, the investor could take the funds and use another wallet to collect all or most of the revenue and intentionally default to get out of payments. A P2P capital market will inherently be a reputation game (for both the business and the bank), so the business will have an incentive to present public addresses (and proving they have the private key) to the bank that the bank can verify are/have been in commercial use. This process might be analogous to TBD's decentralized identity model.
- Once the bank (still anonymously) verifies the reputation/honesty of the business, they can present the prospectus to investors with the multisig wallet script assuring them of both honest, full payments and their capital traunch's place in line.

This does not eliminate the risk of default due to bankrupcy. I think the most logical way to handle those scenarios are for the bank to establish other contracts / multisigs independently with the business and with the investor who voluntarily assume the associated risks. The business might agree to have his banking reputation downgraded by a certain amount, and the bank might promise to cover investors' missed payments up to a certain amount, with a tradeoff on yield.

By no means is this a perfect solution but it might be a step in the right direction.

**Comment by nicolasburtey** [comment]
we want to add lender/borrowing as part of our platform... eventually (in some years, probably not before 18 months) for the reason you are highlighting here. 

I think the first useful features a bitcoin bank currently is the loan of USD/fiat given BTC as a collateral, because this is the easiest to manage on the banking side and doesn't involve much risk for the bank.

The next set of features could be around USD/fiat loan. But there are ton of challenges for underwriting if it's under/uncollateralized.

I think the loan of BTC will be the last one to take off in a bitcoin banking environment, because of the volatility. No SME can afford volatility on a BTC assets. The only type of entity that currently wants to borrow BTC are hedge funds, and we've seen already how this didn't ended up well for many of them.

When the volatility of BTC will be low, then it's worth having some BTC borrow option for SME, BTC which would comes from other user that maybe have put some money on a "BTC saving account" where redeemability is not instant, as a way to limit the duration mismatch issue.

**Reply by humansinstitute** [reply]
My view here isn't that the loan is denominated in btc, but instead the use of bitcoin as rails allows a wider audience to fund the capital of ht loan. 

In this instance the loan would be extended, denominated and repaid in USD (Stablesats) with the flow roughly as: 

1. Bank matches borrower and lending
2. Lender deposits sats (globally) to Bank that are converted into USD (stablesats) 
3. Lender account receives USD - used for business
4. Pays yield to bank -> bank forwards to lender 
5. Loan closed and stable sats paid to bank and loan repaid to lender. 

Here the lender is taking exchange rate risk in the loan as they start with BTC (in order to send abroad for the project), however, if we imagine that the lender also has a community bank and stable sats. Well now they can run the whole loan denominated in USD with no currency risk :)

**Reply by nicolasburtey** [reply]
is the loan collateralized here?

**Reply by humansinstitute** [reply]
No - the idea here would be that the loan is really an investment contract mediated by the community bank. 

The bank finds local businesses that are seeking loans and are willing to pay an interest rate on the loan. 

The best mental model I think is imagining this as for profit version of philanthropic micro-lending. There is a high risk of default against the loan if it's purely peer to peer. But peer to community bank to community business changes the trust model a little. 

In this instance the local business being funded is on the hook to a bank inside that community, the community bank itself is staking the reputation of its community on repaying its loans in order to get access to a wider base of capital globally. 

They certainly can run away with the money, but at that point they are likely to forgo all future investments into the community. The wiser community bank would be encouraged to choose stronger businesses for funding and ensure that investors are repaid.
